TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robotSubSystems/SubSystemManager.java

```diff
@@ -53,15 +59,20 @@ public static ElevatorStates getElevatorStateFromSecondDriver(Gamepad gamepad2)
                         : gamepad2.x ? ElevatorStates.MID : gamepad2.y ? ElevatorStates.HIGH : null;
     }
 
+
     private static void setSubsystemToState(Gamepad gamepad1, Gamepad gamepad2) {
+        float currentTime = (float) subSystemManagerTime.milliseconds();
```

Comment:
If you are using the currentTime in more then 1 place(robot,subsystem manager), you should move it to globalData, and then update it in robot.

Reply:
actually there is no nedd of this, I will remove anything associated with this